### PR TITLE
GCW-3131-Add mixed condition

### PR DESCRIPTION
**Pull request by abulsayyad123**
### Ticket Link: 

https://jira.crossroads.org.hk/browse/GCW-3131

### What does this PR do?

FEATURE: Add a new category called "Mixed" in `donor_condition` drop down on receive-package page and item details page remains same with previous 4 drop down menus.

### CODE REVIEW

File: app/routes/receive_package.js [hunk 1]
```diff
@@ -6,7 +6,10 @@ export default AuthorizeRoute.extend({
   store: Ember.inject.service(),
 
   model(params) {
-    return this.store.findRecord("package", params.package_id);
+    return Ember.RSVP.hash({
+      package: this.store.findRecord("package", params.package_id),
+      donorConditions: this.store.query("donor_condition", { for: "package" })
```
Comment (patrixr) [inline app/routes/receive_package.js:11]: `donorConditions` can perhaps be added to the preload routine, right ?

File: app/routes/review_item/supervisor_messages.js [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
-    this.render('message_template', {controller: 'review_item.supervisor_messages'});
+    this.render("message_template", {
+      controller: "review_item.supervisor_messages"
+    });
   },
 
   afterModel() {
-    var itemId = this.modelFor('review_item').get('id');
-    this.store.query('version', { item_id: itemId, for_item: true });
+    var itemId = this.modelFor("review_item").item.get("id");
```
Comment (patrixr) [inline app/routes/review_item/supervisor_messages.js:11]:
```suggestion
    const itemId = this.modelFor("review_item").get("item.id");
```

Comment (patrixr) [inline app/routes/review_item/supervisor_messages.js:11]:
```suggestion
    const itemId = this.modelFor("review_item").get("item.id");
```

Also:
1. why are we fetching versions ?
2. why are we fetching versions _without_ awaiting the response.  Looks odd to me

I know you didn't write this code, but we should double check. This looks dodgy

**Comment by abulsayyad123** [comment]
Closing this PR due to some fields changes in Donor Condition table and shifted logic to https://github.com/crossroads/shared.goodcity/pull/283

**Pull request closed without merging**